### PR TITLE
add packge sd_logic

### PR DESCRIPTION
**Pull request by zevbo**

### CODE REVIEW

File: packages/sd_logic/sd_logic.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Robot State!"
+description: """
+
+Longer description: https://github.com/zevbo/RobotState
```
Comment (kit-ty-kate) [inline packages/sd_logic/sd_logic.0.1/opam:6]: Would it be possible to have more meaningful `synopsis` and `description` here? It is hard to understand what the package does from either one of them.

Comment (zevbo) [reply packages/sd_logic/sd_logic.0.1/opam:6]: Yeah sure no problem

Comment (zevbo) [reply packages/sd_logic/sd_logic.0.1/opam]: I just added a synopsis and description. Let me know if it's good or it needs more/to be editted :)

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the improvements to the opam file in the source repository so they can be present for the next release?

**Pull request merged**